WooCommerce/Classes/ViewRelated/Inbox/InboxNoteRow.swift

```diff
@@ -5,46 +5,67 @@ import Yosemite
 struct InboxNoteRow: View {
     let viewModel: InboxNoteRowViewModel
 
+    // Tracks the scale of the view due to accessibility changes.
+    @ScaledMetric private var scale: CGFloat = 1
+
     var body: some View {
         VStack(spacing: 0) {
-            VStack(alignment: .leading,
-                   spacing: Constants.verticalSpacing) {
+            VStack(alignment: .leading, spacing: 0) {
                 // HStack with type icon and relative date.
-                // TODO: 5954 - type icon and relative date
+                HStack {
+                    Circle()
+                        .frame(width: scale * Constants.typeIconDimension, height: scale * Constants.typeIconDimension, alignment: .center)
+                        .foregroundColor(Color(Constants.typeIconCircleColor))
+                        .overlay(
+                            viewModel.typeIcon
+                                    .resizable()
+                                    .scaledToFit()
+                                .padding(scale * Constants.typeIconPadding)
+                        )
+                    Text(viewModel.date)
+                        .font(.subheadline)
+                        .foregroundColor(Color(Constants.dateTextColor))
+                    Spacer()
+                }
 
-                // Title.
-                Text(viewModel.title)
-                    .bodyStyle()
-                    .fixedSize(horizontal: false, vertical: true)
+                Spacer()
+                    .frame(height: 8)
```

Comment:
nit: can we use a constant here (from `Constants` enum)?

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/6172/commits/27993444f486ea6ee11e522398b03340e1e67a34